spec/features/admin_split_decision_create_spec.rb

```diff
@@ -36,6 +36,12 @@
     end
 
     expect(split_page).to be_displayed
+    expect(split_page).to have_content "Queued decision"
+
+    Delayed::Worker.new.work_off
+
+    split_page.load split_id: split.id
+    expect(split_page).to be_displayed
 
     expect(Assignment.where(variant: "hammer_time").count).to eq 0
     expect(Assignment.where(variant: "touch_this").count).to eq 5
```

Comment:
We're using the `:delayed_job` adapter in all environments. I think that's saner than the rails default of using a test adapter which tracks enqueued jobs and can potentially run them inline. I also am unsure of how that test adapter behaves in feature specs. This approach felt clean to me: enqueue jobs same as ever, then choose to work them off in order to observe their side effects.